### PR TITLE
Update readme: orientation, release branch, env vars

### PR DESCRIPTION
**Pull request by gabalafou**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,16 +115,26 @@ These are the concrete steps to follow to move your code from branch to branch:
...
+     especially if your pull request is relatively small, in order to keep a
+     clean commit history.
+4. When you want your code change to go live, cut a release branch from
+   `develop`, then open a pull request to merge the release branch into `main`.
+   From the command line:
+   ```sh
+   git checkout develop
+   git pull
+   git checkout -b release-YYYYMMDD
+   git push
```
Comment (bskinn) [inline README.md:127]: Have to link the remote in the push

```suggestion
   git push -u origin release-YYYYMMDD
```

**Review by bskinn — CHANGES_REQUESTED** [review]
@gabalafou  Anyone conversant in git will probably not need this detail, but I think it'd be wise to include.

File: README.md [hunk 2]
```diff
@@ -364,28 +396,9 @@ This repo contains two projects (websites): Consulting and Labs. You must create
...
-If you cannot use the Vercel CLI, you can get the needed environment variables
-from the admin portal:
-
-- [consulting environment
-  variables](https://vercel.com/quansight/quansight-consulting/settings/environment-variables)
-- [labs environment
-  variables](https://vercel.com/quansight/quansight-labs/settings/environment-variables)
+You will need to ask [@gabalafou](https://github.com/gabalafou) for the
+environment variables. [Open Slack direct
+message](slack://user?team=T8XM7KN5B&id=U02JA56KDDY).
```
Comment (gabalafou) [inline README.md:401]:
```suggestion
environment variables. <a href="slack://user?team=T8XM7KN5B&id=U02JA56KDDY">Open Slack direct
message</a>
```

**Review by bskinn — APPROVED** [review]
All seems ok to me with the new changes.

**Pull request merged**